custom/icds_reports/tests/outputs/agg_ccs_record_2017-05-01_4_sorted.csv

```diff
@@ -1,8 +1,8 @@
 state_id,district_id,block_id,supervisor_id,awc_id,month,ccs_status,trimester,caste,disabled,minority,resident,valid_in_month,lactating,pregnant,thr_eligible,rations_21_plus_distributed,tetanus_complete,delivered_in_month,anc1_received_at_delivery,anc2_received_at_delivery,anc3_received_at_delivery,anc4_received_at_delivery,registration_trimester_at_delivery,using_ifa,ifa_consumed_last_seven_days,anemic_normal,anemic_moderate,anemic_severe,anemic_unknown,extra_meal,resting_during_pregnancy,bp1_complete,bp2_complete,bp3_complete,pnc_complete,trimester_2,trimester_3,postnatal,counsel_bp_vid,counsel_preparation,counsel_immediate_bf,counsel_fp_vid,counsel_immediate_conception,counsel_accessible_postpartum_fp,has_aadhar_id,aggregation_level,valid_all_registered_in_month,institutional_delivery_in_month,lactating_all,pregnant_all,valid_visits,expected_visits,state_is_test,district_is_test,block_is_test,supervisor_is_test,awc_is_test
-st1,d1,b1,s1,All,2017-05-01,lactating,All,All,All,All,All,19,19,0,19,5,0,2,0,0,0,0,3.0000000000000000,0,0,0,0,1,18,0,0,1,1,2,0,0,0,4,0,0,0,0,0,0,13,4,19,1,19,0,0,29,0,0,0,0,0
+st1,d1,b1,s1,All,2017-05-01,lactating,All,All,All,All,All,19,19,0,19,5,0,2,0,0,0,0,3.0000000000000000,0,0,0,0,1,18,0,0,1,1,2,0,0,0,4,0,0,0,0,0,0,13,4,19,1,19,0,2,29,0,0,0,0,0

```

Comment:
These test changes do not make a lot of sense. Why are valid visits increasing as a result of no longer double counting?

Reply:
Hey, This is because I have added new rows to pnc fixture to add test for this query change.
@calellowitz